### PR TITLE
feature: this

### LINKED ISSUE #202: this widget

### PR DESCRIPTION
**Pull request by 27moon**
1. **Summary:** _What changed, and why?_
this widget
test for it
added it on dashboard
edited routes for it
2. **Linked Issue:** Closes #202 

3. **UI Screenshots:** If applicable
Снимок экрана 2026-03-27 в 01 45 37

---

## Definition of Done (DoD)

Before requesting review, confirm that:

- [x] Acceptance criteria are satisfied
- [x] Code follows the team's style guide and conventions
- [x] No type assertions are used
- [x] No explicit `any` types are used
- [x] No debug logs or commented-out code remain
- [x] ESLint and formatting checks pass
- [x] No console errors or warnings
- [x] Tests were added or updated if necessary
- [ ] All existing tests pass locally
- [x] Edge cases and error states are handled
- [x] Pull request is linked to an issue

### CODE REVIEW

File: frontend/src/components/features/widgets/This/This.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+          <div className={`${classes.explanation} ${isAnswered ? classes.explanationActive : ''}`}>
+            <Text fw={800} size="xs" c="brand" mb={4} tt="uppercase">
+              {t('widgets.this_quiz.explanation')}
+            </Text>
+            <div className={classes.explanationScrollArea}>
+              <Text size="sm" lh="md">
+                {explanation}
+              </Text>
+            </div>
+          </div>
```
Comment (oneilcode) [inline frontend/src/components/features/widgets/This/This.tsx]: Предлагаю использовать теги Мантин
